RUMM-2490 Add package for codepush support (#261)
DataDog/dd-sdk-reactnative: packages/codepush/README.md

# React-Native Monitoring for AppCenter CodePush

## Setup

> **Note**: This package is an integration for the [`react-native-code-push`][1] library, please make sure you first install and setup the core [`mobile-react-native`][2] SDK.

Reviewer: ```suggestion
**Note**: This package is an integration for the [`react-native-code-push`][1] library. Before using it, you must first install and setup the core [`mobile-react-native`][2] SDK.
```
